pkg/console/operator/status.go

```diff
@@ -318,3 +319,15 @@ func (c *consoleOperator) ConditionsManagementStateRemoved(operatorConfig *opera
 
 	return operatorConfig
 }
+
+func (c *consoleOperator) UpgradableWithUnsupportedConfigOverrides(operatorConfig *operatorsv1.Console, message string) *operatorsv1.Console {
```

Comment:
This would be better inlined. I don't see a good reason to wrap a one-liner with a helper, and it obscures what it's setting. We're only calling it in one place.

The helpers make more sense to me if they're setting several related conditions together, but not in the case where it's setting a single condition.

Reply:
refact the status setting in the operator (as we already discussed) and probably use the `v1helpers.SetOperatorCondition` method directly.

Reply:
TL;DR its not ideal, but inconsistent seems worse than the correct fix. Lets fix it the right way.

We should not refactor all of the status settings in this PR, that should be done as a separate tech debt PR (as I've said before, I agree, although strongly that it should be factored out into one place -- return enough structures from `sync_v400()` to handle status once, without breaking the fail-early mechanism.

I hesitate to introduce **inconsistencies** before then and just make it messier.  This is really just a simple `factory` function.

Reply:
Eliminating this entirely in my next push, will have `NewUnsupportedConfigOverridesController()` handle the status reportin.